doc/sphinx/practical-tools/utilities.rst

```diff
@@ -34,6 +34,13 @@ For example, to statically link |Ltac|, you can just do:
 
 and similarly for other plugins.
 
+Building a |Coq| project
+------------------------
+
+As of today it is possible to build Coq projects using two tools:
+
+- coq_makefile, which is distributed by Coq and it is based on generating a makefile,
+- Dune, the standard OCaml build tool which since version 1.9 does support building of Coq libraries.
 
 Building a |Coq| project with coq_makefile
 ------------------------------------------
```

Comment:
I suggest making this a subsection (and the Dune part too).

```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

This will also require changing the sub-sections of this section to use the level of title below.

Reply:
How can we change that level?